### PR TITLE
feat: Cart created

### PR DESCRIPTION
**Pull request by ShoaibTamboli**
Cart feature created.

### CODE REVIEW

**Comment by athar05** [comment]
Looks good to merge

File: pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    flex-direction: column;
+    width: 30%;
+    background-color: var(--slategray);
+    padding-left: 15px;
+    justify-content: space-evenly;
+}
+
+.price-detail-spacing{
+    display: flex;
+    justify-content: space-around;
```
Comment (sumitch29) [inline pages/cart/cart.css:34]: give justify-content: space-between and padding from right, then it will look good.

**Review by sumitch29 — APPROVED** [review]
Make button size equal for add to wishlist and remove from cart. Otherwise looks good.
use rem above 10px.

File: pages/cart/cart.css (hunk 1 above)
Comment (ShoaibTamboli) [reply pages/cart/cart.css:34]: ok will do

**Pull request merged**